src/Content/BoardPage/AddListButton/AddListButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from 'react';
+import style from './AddListButton.module.css';
+import { AddListButtonPropsType } from '../../../types/types';
```

Comment:
This is a huge mess, prop types are usually defined within the same component. No biggie though.